README.md

```diff
@@ -130,3 +130,20 @@ like "dieses Wort wird verwendet" ("this word is used").
   not "克林貢" which is used in Taiwanese Mandarin (and which is returned by
   Google Translate for "Traditional Chinese").
 
+### Finnish
+
+- Translations need not to be direct translations from TKD entries, but later
+  clarifications and additions must be considered when choosing the Finnish word.
+  The Finnish translation may have less or more words than the English one, if
```

Comment:
s/less/fewer

Because words are countable, though the distinction between "less" and "fewer" is disappearing in English.